index.html

```diff
@@ -9,10 +9,6 @@
     <link rel="stylesheet" href="styles2.css">
     <title>My Portfolio</title>
```

Comment:
- [ ] Kindly make sure your page title adequately describe your page, not a generalized page title. 
 - Poor titles:
   - Microverse project | Your portfolio.
   - Portifolio project

 - Better page titles:
   - Mark's portifolio
   - Mark full-stack webdeveloper

Reply:
done.